### PR TITLE
DB migrations

### PR DESCRIPTION
**Pull request by liubamiron**

### CODE REVIEW

File: 2021_12_12_084922_create_benefits_table.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('benefits', function (Blueprint $table) {
+            $table->id();
+            $table->enum('name', ['WiFi', 'Garage', 'Autonomous heat sources']);
```
Comment (egorbwork) [inline 2021_12_12_084922_create_benefits_table.php]: Doesn't have sense to have enum, you are making this list not extensible

File: 2021_12_12_103026_alter_imobils_table_add_seria_column.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        //
```
Comment (egorbwork) [inline 2021_12_12_103026_alter_imobils_table_add_seria_column.php]: rollback is missing

**Comment by egorbwork** [comment]
Is missing alter for column change and for column removal.

File: 2021_12_10_000000_create_owners_table.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        DB::statement("DROP TABLE `owners`;
```
Comment (egorbwork) [inline 2021_12_10_000000_create_owners_table.php:27]: Please fix indentation.

Comment (egorbwork) [reply 2021_12_10_000000_create_owners_table.php:27]: in all places

File: 2021_12_14_001023_alter_imobils_table_remove_column.php [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('imobils', function (Blueprint $table) {
+            $table->string('description')->after('balcony');
```
Comment (egorbwork) [inline 2021_12_14_001023_alter_imobils_table_remove_column.php]: description was not a string type

**Review by egorbwork — CHANGES_REQUESTED** [review]

File: 2021_12_14_001023_alter_imobils_table_remove_column.php (hunk 4 above)
Comment (liubamiron) [reply 2021_12_14_001023_alter_imobils_table_remove_column.php]: change string into text  https://github.com/liubamiron/bootcamp-project/pull/4/commits/c8fbf87664efd5eb65043210442d39435136f0dd

File: 2021_12_12_084922_create_benefits_table.php (hunk 1 above)
Comment (liubamiron) [reply 2021_12_12_084922_create_benefits_table.php]: change enum into string, and unique

File: 2021_12_12_103026_alter_imobils_table_add_seria_column.php (hunk 2 above)
Comment (liubamiron) [reply 2021_12_12_103026_alter_imobils_table_add_seria_column.php]: https://github.com/liubamiron/bootcamp-project/pull/4/commits/c9ceb908511d6cbce68f5baf0878b232d4bcba57

File: 2021_12_10_000000_create_owners_table.php (hunk 3 above)
Comment (liubamiron) [reply 2021_12_10_000000_create_owners_table.php:27]: fixed indentation

File: 2021_12_12_084922_create_benefits_table.php (hunk 1 above)
Comment (liubamiron) [reply 2021_12_12_084922_create_benefits_table.php]: https://github.com/liubamiron/bootcamp-project/commit/c8fbf87664efd5eb65043210442d39435136f0dd

File: 2021_12_10_000000_create_owners_table.php (hunk 3 above)
Comment (egorbwork) [reply 2021_12_10_000000_create_owners_table.php:27]: Not in all places.

**Review by egorbwork — APPROVED** [review]

**Pull request merged**